forex_python/converter.py

```diff
@@ -25,21 +25,23 @@ def __init__(self, force_decimal=False):
         self._force_decimal = force_decimal
 
     def _source_url(self):
-        return "https://theforexapi.com/api/"
+        return "https://api.exchangerate.host/"
```

Comment:
why not allow specying source url as parameter?

Reply:
Most apis are not going to be this similar in terms of api surface. This one is 95% there so still requires minor changes. I don't think having the provider url as a parameter makes sense unless theforexapi.com comes back online again. 

If supporting multiple apis is required there needs to be some refactoring to abstract the api provider from the currency class.

Reply:
It makes sense, you can host your own api since theforexapi is open source, there should be an option to override the url...